shap/benchmark/metrics.py

```diff
@@ -73,298 +74,298 @@ def score_function(X_train, X_test, y_train, y_test, attr_function, trained_mode
             X_train, y_train, X_test, y_test, attr_function(X_test),
             model_generator, score_map, trained_model
         )
-    return None, __score_method(X, y, None, model_generator, score_function, method_name)
-
-def consistency_guarantees(X, y, model_generator, method_name):
-    """ Consistency Guarantees
-    transform = "identity"
-    sort_order = 1
-    """
-
-    # 1.0 - perfect consistency
-    # 0.8 - guarantees depend on sampling
-    # 0.6 - guarantees depend on approximation
-    # 0.0 - no garuntees
-    guarantees = {
-        "linear_shap_corr": 1.0,
-        "linear_shap_ind": 1.0,
-        "coef": 0.0,
-        "kernel_shap_1000_meanref": 0.8,
-        "sampling_shap_1000": 0.8,
-        "random": 0.0,
-        "saabas": 0.0,
-        "tree_gain": 0.0,
-        "tree_shap_tree_path_dependent": 1.0,
-        "tree_shap_independent_200": 1.0,
-        "mean_abs_tree_shap": 1.0,
-        "lime_tabular_regression_1000": 0.8,
-        "lime_tabular_classification_1000": 0.8,
-        "maple": 0.8,
-        "tree_maple": 0.8,
-        "deep_shap": 0.6,
-        "expected_gradients": 0.6
-    }
-
-    return None, guarantees[method_name]
+    return None, __score_method(X, y, None, model_generator, score_function, explainer)
+
+# def consistency_guarantees(X, y, model_generator, method_name):
```

Comment:
remove commented code?  but I thought consistency guarantees was useful, any reason we are removing this now?

Reply:
This was originally commented out as we are temporarily leaving it out for the new API, but old files will generally be removed later so it will for now be uncommented